prime-router/src/test/kotlin/ReceiverTests.kt

```diff
@@ -43,6 +44,7 @@ internal class ReceiverTests {
             topic = Topic.FULL_ELR,
             customerStatus = CustomerStatus.INACTIVE,
             conditionFilter = listOf("%testPerformedCodes.intersect('123-0'|'600-7').exists()"),
+            mappedConditionFilter = listOf(CodeStringConditionFilter("123-0,600-7")),
```

Comment:
Again, feel like this needs to be a separate test without the conditionFilter set

Reply:
Nice catch 👍 good idea